posthog/temporal/data_imports/external_data_job.py

```diff
@@ -247,7 +247,7 @@ async def run(self, inputs: ExternalDataWorkflowInputs):
             )
 
             timeout_params = (
-                {"start_to_close_timeout": dt.timedelta(weeks=1), "retry_policy": RetryPolicy(maximum_attempts=1)}
+                {"start_to_close_timeout": dt.timedelta(weeks=1), "retry_policy": RetryPolicy(maximum_attempts=3)}
```

Comment:
Can you do this conditional on whether it's incremental or not (`incremental` should be in scope). The whole workflow restarts up to three times too anyway

Reply:
I don't understand, what would be different if this is `incremental` or not? Why did we have less attempts before?

Reply:
But that was already the case? I am not touching retries for full refresh jobs?

Reply:
OOHHH, I didn't see the `if incremental` on the line below, apologies!